kerchunk/hdf.py

```diff
@@ -189,12 +189,40 @@ def _transfer_attrs(
                     f"TypeError transferring attr, skipping:\n {n}@{h5obj.name} = {v} ({type(v)})"
                 )
 
+    def _decode_blosc(properties): # 32001
+        blosc_compressors = ('blosclz', 'lz4', 'lz4hc', 'snappy', 'zlib', 'zstd')
+        _1, _2, bytes_per_num, total_bytes, clevel, shuffle, compressor = properties
+        return dict (id="blosc", blocksize=total_bytes, clevel=clevel, shuffle=shuffle, cname=blosc_compressors[compressor],)
+
+    def _decode_zstd(properties): #32015
+        return dict (id='zstd', level=properties[0],)
+
+
+    decoders = { "32001" : _decode_blosc,
+                 "32015" : _decode_zstd,
+                }
+
+    def _decode_filters(self, h5obj: Union[h5py.Dataset, h5py.Group]):
+        if len(h5obj._filters.keys()) > 1:
+            raise RuntimeError(
+                f"{h5obj.name} uses multiple filters {list (h5obj._filters.keys())}. This is not supported by kerchunk."
```

Comment:
This came up in https://github.com/fsspec/kerchunk/issues/342 - actually zarr *does* support multiple filters, except only the final ne is called "compression"